src/state/slices/txHistorySlice/selectors.ts

```diff
@@ -65,10 +66,13 @@ export const selectTxById = createCachedSelector(
 export const selectTxDateByIds = createDeepEqualOutputSelector(
   selectTxIdsParam,
   selectTxs,
-  (txIds: TxId[], txs) =>
-    txIds
-      .map((txId: TxId) => ({ txId, date: txs[txId].blockTime }))
-      .sort((a, b) => b.date - a.date),
+  (txIds: TxId[], txs) => {
+    return txIds
+      .map((txId: TxId) => {
+        return { txId, date: txs[txId].blockTime }
+      })
+      .sort((a, b) => b.date - a.date)
+  },
```

Comment:
makes debugging possible

Reply:
q: while the braces over function return shorthand are indeed goat for debugger purposes, do we want to revert it before merging to keep blame clean?